src/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs

```diff
@@ -378,6 +378,19 @@ object ICollection.SyncRoot
             }
         }
 
+        int ICollection.Count 
```

Comment:
I had the same reaction about these explicit interfaces and chatted with @weshaggard about it. We decided to move them over since they are part of netstandard 2.0.

They don't add any value over their implicitly implemented counterparts but they help us reach parity with the current definition of netstandard2.0